Xamarin.Essentials/Screenshot/Screenshot.watchos.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.IO;
+using System.Threading.Tasks;
+using Foundation;
+using UIKit;
+
+namespace Xamarin.Essentials
+{
+    public static partial class Screenshot
+    {
+        internal static bool PlatformCanCapture => false; // On WatchOs 3 this appears to be possible if activated through settings.
```

Comment:
Interesting... Can we detect this value or read it? Then we can have this property return true sometimes.

Reply:
Perhaps, but AFAIK it is not possible currently to take screenshots on watchos programatically. Would make no sense returning true then IMHO

Reply:
Good enough.